src/main.rs

```diff
@@ -0,0 +1,26 @@
+#![feature(proc_macro_hygiene, decl_macro)]
+
+#[macro_use]
+extern crate rocket;
+
+mod command;
+mod database;
+mod server;
+
+use clap::Clap;
+use command::Command;
+
+#[derive(Clap)]
+pub struct Opts {
+    #[clap(short, long, default_value = "database.yaml")]
+    database: String,
+
+    #[clap(subcommand)]
+    pub command: Command,
```

Comment:
`pub` が消せる気がする(間違ってたらすまない
```suggestion
    command: Command,
```

Reply:
6f52936